ddtrace/settings/config.py

```diff
@@ -202,14 +206,17 @@ def trace_headers(self, whitelist):
         self.http.trace_headers(whitelist)
         return self
 
-    def header_is_traced(self, header_name):
```

Comment:
this API is public, I don't think we can get rid of it

Reply:
I added back the public method so we do not break the compatibility.